src/platform/linux/x11.rs

```diff
@@ -23,7 +23,6 @@ use std::{
 	thread::{self, JoinHandle},
 	thread_local,
 	time::{Duration, Instant},
-	usize,
```

Comment:
This import was causing a lint error, see: https://github.com/warpdotdev/arboard/actions/runs/9845014311/job/27179645023.

Not sure why this is being triggered on this PR and not any previous ones, but removing the import fixes it and I don't see it having any tangible impact.